local_tests/test_models.py

```diff
@@ -2,16 +2,20 @@
 import io
 import os
 import re
+import signal
 import sys
 import subprocess
+import traceback
+
+from side_by_side import print_side_by_side
 
 def color_text(s, color):
     return colored.stylize(s, colored.fg(color))
     
-FNULL = open(os.devnull, 'w')
+stderr_file_name = 'err.out.txt'
```

Comment:
uhhh err.out.txt? Why not just err.out or err.txt?

Reply:
ok